.github/workflows/integration-test.yml

```diff
@@ -128,6 +128,12 @@ jobs:
         component: ${{ fromJson(needs.list.outputs.workflow_matrix) }}
 
     steps:
+    - name: "Remove Android SDK and .NET"
```

Comment:
add comment as to why this step is being performed

Reply:
Done!